api/pkg/transformer/pipeline/table_transform_op.go

```diff
@@ -18,6 +19,9 @@ func NewTableTransformOp(tableTransformSpec *spec.TableTransformation) Op {
 }
 
 func (t TableTransformOp) Execute(context context.Context, env *Environment) error {
+	span, _ := opentracing.StartSpanFromContext(context, "pipeline.TableTransformOp")
+	defer span.Finish()
+
 	inputTableName := t.tableTransformSpec.InputTable
 	outputTableName := t.tableTransformSpec.OutputTable
```

Comment:
We can add more data to span by using SetTag:

```
span.SetTag("table.input", inputTableName)
span.SetTag("table.output", outputTableName)
```

Having this can help us identify the operation on an individual level, especially when there are multiple similar operations.